File: scholar-business/src/main/java/com/varun/scholar/business/data/entities1/PayrollSearch.java
package com.varun.scholar.business.data.entities1;

import java.util.Date;

import javax.persistence.Entity;
import javax.persistence.Temporal;
import javax.persistence.TemporalType;

import com.varun.scholar.business.data.entities.AbstractEntity;

@Entity
public class PayrollSearch extends AbstractEntity {
	String userName;
	String userRole;
	@Temporal(TemporalType.DATE)
	Date dateOfJoining;
	String grade;
	String experience;
	@Temporal(TemporalType.DATE)
	Date lastChange;
}

File: scholar-web/src/main/webapp/app/model/finance/payroll/PayrollSearch.js
Ext.define('scholar.model.finance.payroll.PayrollSearch', {
	extend: 'Ext.data.Model',
	fields : [  {
		name : 'userName',
		type : 'string'
	},
	{
		name : 'userRole',
		type : 'string'
	},
	{
		name : 'dateOfJoining',
		type : 'date',
		dateFormat : 'Y-m-d'
	},
	{
		name : 'grade',
		type : 'string'
	},
	{
		name : 'experience',
		type : 'string',
	},
	{
		name : 'lastChange',
		type : 'date',
		dateFormat : 'n/j h:ia'
	} ]
	
});